iothub/device/samples/PnpDeviceSamples/TemperatureController/SimpleTemperatureControllerSampleNew.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    public class SimpleTemperatureControllerSampleNew
+    {
+        private static readonly Random s_random = new();
+        private static readonly PayloadConvention s_systemTextJsonPayloadConvention = new SystemTextJsonPayloadConvention();
```

Comment:
To make it simple we should probably not show how to use the convention, we have other samples that do that.

Reply:
good point, I'll remove it for the "simple" sample